docs/en/pid-eaa-issuance.rst

```diff
@@ -137,7 +137,7 @@ The JWS payload of the request object is represented below:
         "type":"openid_credential",
         "format": "vc+sd-jwt",
         "credential_definition": {
-            "type": ["PersonIdentificationData"]
+            "type": "PersonIdentificationData"
         }
     }
     ],
```

Comment:
@fmarino-ipzs

`authorization_details` is an array, and each credential authorization is a json object, then requesting multiple credentials is possible by specifing type, format and credential definition.

fair enough I'd say